src/NUnitFramework/framework/Constraints/Comparers/EquatablesComparer.cs

```diff
@@ -71,7 +88,7 @@ private static MethodInfo FirstImplementsIEquatableOfSecond(Type first, Type sec
             return pair.Value;
         }
 
-        private static IList<KeyValuePair<Type, MethodInfo>> GetEquatableGenericArguments(Type type)
+        private static List<KeyValuePair<Type, MethodInfo>> GetEquatableGenericArguments(Type type)
```

Comment:
`List<T>.GetEnumerator()` returns a value type, which avoid an allocation at the caller.